src/kajol/FindCountSumAndAverageofPrimeNumbers.java

```diff
@@ -0,0 +1,40 @@
+package kajol;
+
+import java.util.Scanner;
+
+public class FindCountSumAndAverageofPrimeNumbers {
+	
+	void findPrimeNumbers(int start,int end) {
+		int count=0;int sum=0;
+		float average=0;
+		System.out.println("Prime Numbers Between "+start+" and "+end+" :");
+		for(int index=start;index<=end;index++) {
+			boolean flag=true;
+			for(int innerIndex=2;innerIndex<=Math.sqrt(index);innerIndex++) {
+				if(index%innerIndex==0)
+					flag=false;
+			}
+			if(flag) {
+				count++;
+				System.out.print(index+" ");
+				sum=sum+index;
+				average=sum/count;
```

Comment:
this average formula you can do after for loop iterations, at end of loop we need to check this operation. dont'need to check avg on every iteration.